src/00_utils.js

```diff
@@ -0,0 +1,76 @@
+function pushPoly(array, item) {
+  array[array.length] = item;

```

Comment:
I'd recommend you do not mutate functions arguments and to keep functions clean, you can define a new variable
https://habr.com/ru/articles/437512/